pkg/kubelet/container_bridge.go

```diff
@@ -43,15 +43,16 @@ func createCBR0(wantCIDR *net.IPNet) error {
 		glog.Error(err)
 		return err
 	}
-	// restart docker
+	// Stop docker so that babysitter process can restart it again with proper configurations and
+	// checkpoint file
 	// For now just log the error. The containerRuntime check will catch docker failures.
 	// TODO (dawnchen) figure out what we should do for rkt here.
 	if util.UsingSystemdInitSystem() {
-		if err := exec.Command("systemctl", "restart", "docker").Run(); err != nil {
+		if err := exec.Command("systemctl", "stop", "docker").Run(); err != nil {
```

Comment:
I feel this is risky because you're leaving the node in a broken state. Aren't there setups where no supervisord is running?